modules/internal/Bytes.chpl

```diff
@@ -1097,11 +1097,29 @@ module Bytes {
      :returns: A new :mod:`bytes <Bytes>` which is the result of repeating `s`
                `n` times.  If `n` is less than or equal to 0, an empty bytes is
                returned.
+          
+     The operation is commutative.
+     For example:
+
+     .. code-block:: chapel
+        
+        writeln(b"Hello! "*3);
+        or
+        writeln(3*b"Hello! ");
+
+     Results in::
+
+        Hello! Hello! Hello!         
   */
   operator bytes.*(s: bytes, n: integral) {
     return doMultiply(s, n);
   }
 
+  pragma "no doc"
+  operator bytes.*(n: integral, s: bytes) {
```

Comment:
For now, let's convert this and the one above to standalone functions, so they look like:

```
operator *(....)
```

Also pinging @lydia-duncan. The string*integral operator is standalone, but bytes*integral is a method on `bytes`. I think we should make string/bytes consistent in this matter, and if I were to choose, I'd make them standalone with the philosophical argument that all string/bytes/int are built-in types, and as such, operators between them doesn't necessarily belong to one type.

Reply:
okay @e-kayrakli , I understood, will commit the changes soon